app/models/character.rb

```diff
@@ -1,4 +1,6 @@
 class Character < ApplicationRecord
+  acts_as_paranoid
```

Comment:
- constants
- modules
- associations
- validations
- delegations
- gem methods
- class methods
- instance methods

So `acts_as_paranoid` must be near `enum`